<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_NewTerrain</class>
 <widget class="QDialog" name="Dialog_NewTerrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>277</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>402</width>
    <height>277</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>New Terrain</string>
  </property>
  <widget class="QLineEdit" name="input_heightFile">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>41</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Height File:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Width x Height:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_width">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>130</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_height">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>187</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Scale:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_scale">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>68</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
   <property name="text">
    <string>12.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_browseHeightFile">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Ok">
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Cancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Texture:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_textureFile">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>81</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_browseTextureFile">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>button_Ok</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_NewTerrain</receiver>
   <slot>on_button_Ok_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_browseHeightFile</sender>
   <signal>pressed()</signal>
   <receiver>Dialog_NewTerrain</receiver>
   <slot>on_button_browseTerrainFile_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_NewTerrain</receiver>
   <slot>on_button_Cancel_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_browseTextureFile</sender>
   <signal>pressed()</signal>
   <receiver>Dialog_NewTerrain</receiver>
   <slot>on_button_browseTextureFile_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_button_Ok_clicked()</slot>
  <slot>on_button_browseTerrainFile_clicked()</slot>
  <slot>on_button_Cancel_clicked()</slot>
  <slot>on_button_browseTextureFile_clicked()</slot>
 </slots>
</ui>
